pkg/util/util.go

```diff
@@ -117,9 +116,20 @@ func NamespaceKubernetesObjectWithTrim(componentName, applicationName string) (s
 		return value, nil
 	}
 
-	// trim to 31 characters
-	componentName = componentName[:31]
-	applicationName = applicationName[:31]
+	// trim app name to max 31 characters
+	appNameLen := len(applicationName)
+	if appNameLen > 31 {
```

Comment:
Not a big fan of magic numbers like 31 or 62 repeated here :-)
Can you extract those as constants instead? This will make them easily updatable if needed.